app/Platform/Controllers/PlayerCompletionProgressController.php

```diff
@@ -131,6 +131,18 @@ private function useSortedList(array $filteredAndJoinedGamesList, string $sortOr
             });
         }
 
+        if ($sortOrder === 'game_title') {
+            usort($filteredAndJoinedGamesList, function ($a, $b) {
+                return strcmp($a['Title'], $b['Title']);
```

Comment:
Not sure if it matters, but Hacks and Homebrew are sorted after normal games.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/87aca300-d7c0-4d71-a639-eebc86837efd)

Developer sets has some code that generates a sort title and uses that for sorting:
https://github.com/RetroAchievements/RAWeb/blob/f5105523a1fe8b6eb78114a8130f23d77c7b4aee/app/Platform/Controllers/DeveloperSetsController.php#L134-L138

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/05df65e6-33c0-4b25-a2e4-46bfdb6fce59)

Reply:
I think it matters, will take a look.

Reply:
Thanks, resolved in https://github.com/RetroAchievements/RAWeb/pull/2039/commits/e86335d7a95ad48dbcc73d262bb1643cd79d408c.